csv/test-survey2.csv

```diff
@@ -0,0 +1 @@
+user
```

Comment:
The content of the csv dir should not be added (You can add this file in the .gitignore with the other one).